cli/azd/pkg/tools/docker/docker.go

```diff
@@ -90,12 +107,30 @@ func (d *docker) Build(
 
 	args = append(args, buildContext)
 
-	res, err := d.executeCommand(ctx, cwd, args...)
+	// Build and produce output
+	runArgs := exec.NewRunArgs("docker", args...).WithCwd(cwd)
+
+	if buildProgress != nil {
+		// setting stderr and stdout both, as it's been noticed
+		// that docker log goes to stderr on macOS, but stdout on Ubuntu.
+		runArgs = runArgs.WithStdOut(buildProgress).WithStdErr(buildProgress)
+	}
+
+	_, err = d.commandRunner.Run(ctx, runArgs)
 	if err != nil {
 		return "", fmt.Errorf("building image: %w", err)
 	}
 
-	return strings.TrimSpace(res.Stdout), nil
+	// // Build should just return cached img id
```

Comment:
NIT: Let's remove this commented out code that use to call `docker build` with `-q`.